macos/purchases_flutter.podspec

```diff
@@ -0,0 +1,24 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint purchases_flutter.podspec' to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'purchases_flutter'
```

Comment:
I suppose it doesn't matter, since these wouldn't be published anyway, but any concerns about this having the same name as the iOS pods? or is it intentional so that the import is the same?

Reply:
this one was created automatically by Flutter and I didn't have any problems when building so I think we are fine